content/rs/clusters/monitoring/console-metrics-definitions.md

```diff
@@ -18,68 +18,96 @@ aliases: /rs/administering/monitoring-metrics/definitions/
-| Pending writes min | Active-Active | Minimum number of writes queued |  |
-| RAM fragmentation | Database, Shard | Ratio between the used (allocated) memory and the actual physical RAM that is used. |  |
-| Read misses/sec | Database, Shard | Number of read operations (per sec) on non-existing keys | This metric is more relevant for caching use cases. |
-| Reads latency | Database | Latency per read operation | The graph shows average, min, max and last values are also shown |
-| Reads/sec | Database | Number of total reads per second | For example: GET |
+| Other commands latency | Database | Latency per other command | The graph shows average, min, max and last latency values |
+| Outgoing traffic[^1] | Cluster, Node, Database | Total outgoing traffic (in bytes/sec) from the database |  |
+| Pending writes max | Active-Active | Maximum number of write opreations queued |  |
+| Pending writes min | Active-Active | Minimum number of write operations queued |  |
+| RAM fragmentation | Database, Shard | Ratio between the used (allocated) memory and the physical RAM that is used. |  |
+| Read misses/sec[^1] | Database, Shard | Number of read operations (per sec) on non-existing keys |  |
+| Reads latency | Database | Latency per read operation | The graph shows average, min, max and last latency values |
+| Reads/sec | Database | Number of total read operations per second | For example: GET |
 | Total CPU usage | Database, Shard | Percent usage of the CPU |  |
-| Total keys | Database, Shard | Total number of keys in the dataset (not including replication, even if replication enabled) | Calculated as the sum of all keys of all master shards. |
-| Used memory | Database, Shard | Total memory used by the database, including RAM, Flash (if enabled) and replication (if enabled) | **Note** – used memory does not include:<br>1. The fragmentation overhead<br><br>2. The replica replication buffers at the masters<br><br>3. Memory used by Lua scripts<br><br>4. Copy On Write (COW) operation that can be triggered by:<br><br>  - A full replication process is started<br><br>  - A database snapshot process is started<br><br>  - AOF rewrite process is started |
-| Write misses/sec | Database, Shard | Number of write operations (per sec) on non-existing keys | This metric is more relevant for caching use cases. |
+| Total keys[^1] | Database, Shard | Total number of keys in the dataset (not including replication, even if replication enabled) | Calculated as the sum of all keys of all master shards. |
+| Used memory | Database, Shard | Total memory used by the database, including RAM, Flash (if enabled) and replication (if enabled) | Does not include:<ol><li>Fragmentation overhead</li><li>Replica replication buffers at the primary nodes</li><li>Memory used by Lua scripts</li><li>Copy On Write (COW) operation that can be triggered by:<br><ul><li>A full replication process</li><li>A database snapshot process</li><li>AOF rewrite process</li></ul></li></ol> |
```

Comment:
Should the footnote also apply to this one?

Reply:
Yes, it should. Fixed in cb911da.